src/cryptonote_core/service_node_list.cpp

```diff
@@ -1767,6 +1767,9 @@ namespace service_nodes
       return false;
     }
 
+    if (winner == null_block_winner)
```

Comment:
When does this trigger the check below? It doesn't ordinarily because the null block winner has 1 payout entry and the miner tx is allocated an output for the null winner.

In HF16 this does change but the code here is re-written to handle that.

Reply:
It triggered for me trying to spin up the new network without any service nodes registered.

Reply:
Maybe an off-by-one causing problems on the actual fork block or something?  I didn't investigate it in depth.

Reply:
Hmm odd, not sure that's ever triggered for me, in tests as well. I removed the check and can sync current devnet ok

Reply:
It's also possible there was some race condition in setting up the initial mined blocks--difficulty started super lower so it mined a bunch at once.  If it's syncing fine now we can probably drop it safely.

Reply:
It's also possible it was related to the `allow == timestamps` change.